pkg/rpc/server_helper_test.go

```diff
@@ -0,0 +1,182 @@
+package rpc
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/CityOfZion/neo-go/config"
+	"github.com/CityOfZion/neo-go/pkg/core"
+	"github.com/CityOfZion/neo-go/pkg/core/storage"
+	"github.com/CityOfZion/neo-go/pkg/core/transaction"
+	"github.com/CityOfZion/neo-go/pkg/crypto/hash"
+	"github.com/CityOfZion/neo-go/pkg/network"
+	"github.com/CityOfZion/neo-go/pkg/rpc/result"
+	"github.com/CityOfZion/neo-go/pkg/rpc/wrappers"
+	"github.com/CityOfZion/neo-go/pkg/util"
+	"github.com/stretchr/testify/require"
+)
+
+// SendTXResponse struct for testing.
+type SendTXResponse struct {
+	Jsonrpc string `json:"jsonrpc"`
+	Result  bool   `json:"result"`
+	ID      int    `json:"id"`
+}
+
+// ValidateAddrResponse struct for testing.
+type ValidateAddrResponse struct {
+	Jsonrpc string                           `json:"jsonrpc"`
+	Result  wrappers.ValidateAddressResponse `json:"result"`
+	ID      int                              `json:"id"`
+}
+
+// GetPeersResponse struct for testing.
+type GetPeersResponse struct {
+	Jsonrpc string `json:"jsonrpc"`
+	Result  struct {
+		Unconnected []int `json:"unconnected"`
+		Connected   []int `json:"connected"`
+		Bad         []int `json:"bad"`
+	} `json:"result"`
+	ID int `json:"id"`
+}
+
+// GetVersionResponse struct for testing.
+type GetVersionResponse struct {
+	Jsonrpc string         `json:"jsonrpc"`
+	Result  result.Version `json:"result"`
+	ID      int            `json:"id"`
+}
+
+// IntResultResponse struct for testing.
+type IntResultResponse struct {
+	Jsonrpc string `json:"jsonrpc"`
+	Result  int    `json:"result"`
+	ID      int    `json:"id"`
+}
+
+// StringResultResponse struct for testing.
+type StringResultResponse struct {
+	Jsonrpc string `json:"jsonrpc"`
+	Result  string `json:"result"`
+	ID      int    `json:"id"`
+}
+
+// GetBlockResponse struct for testing.
```

Comment:
I'm wondering why are they only used for testing. These should be in the main code.

Reply:
Could be, right. In the main code looks like it's not that much required